src/index.html

```diff
@@ -13,6 +13,47 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
```

Comment:
Please check how class name for block with modifier should look like according to BEM. Here, it's not used as expected for all the `stars` blocks.